src/connectors.rs

```diff
@@ -1178,6 +1178,7 @@ pub(crate) fn builtin_connector_types() -> Vec<Box<dyn ConnectorBuilder + 'stati
         Box::new(impls::http::server::Builder::default()),
         Box::new(impls::otel::client::Builder::default()),
         Box::new(impls::otel::server::Builder::default()),
+        Box::new(impls::gbq::Builder::default()),
```

Comment:
Perhaps namespace this to `impls::gbq::writer::Builder::default()` as this will make adding `reader` orthogonal in due course?